src/Microsoft.Identity.Web/WebAppCallsWebApiAuthenticationBuilderExtensions.cs

```diff
@@ -160,7 +174,7 @@ public static class WebAppCallsWebApiAuthenticationBuilderExtensions
                 options.Events.OnRedirectToIdentityProviderForSignOut = async context =>
                 {
                     // Remove the account from MSAL.NET token cache
-                    var tokenAcquisition = context.HttpContext.RequestServices.GetRequiredService<ITokenAcquisition>() as ITokenAcquisitionInternal;
+                    var tokenAcquisition = (ITokenAcquisitionInternal)context.HttpContext.RequestServices.GetRequiredService<ITokenAcquisition>();
```

Comment:
is this part of a fix for [240](https://github.com/AzureAD/microsoft-identity-web/issues/240)? if so, can it go in a separate PR?

Reply:
No, this is just to make it non-nullable. #240 will change this line anyway.